packages/patternfly-4/react-docs/src/templates/mdxTemplate.js

```diff
@@ -3,15 +3,27 @@ import PropTypes from 'prop-types';
 import { graphql } from 'gatsby';
 import SidebarLayout from './sidebarLayout';
 import { CSSVars, PropsTable, LiveEdit } from '../components/componentDocs';
-import { Title, PageSection } from '@patternfly-safe/react-core';
+import { Title, PageSection, Text, TextList, TextListItem } from '@patternfly-safe/react-core';
 import { MDXProvider } from '@mdx-js/react';
 import { MDXRenderer } from '../components/mdx-renderer';
 
 const components = {
+  wrapper: componentProps => <div className="theme-pf-c-content" {...componentProps} />,
   code: LiveEdit,
-  pre: React.Fragment
+  pre: React.Fragment,
+  p: componentProps => <Text component="p" {...componentProps} />,
```

Comment:
We (eventually) need to make sure every HTML tag is covered here that doesn't look right inside `<div className="theme-pf-c-content">`.